windows/src/buildtools/buildunidata/buildunidata.dproj

```diff
@@ -38,7 +38,7 @@
         <Icon_MainIcon>$(BDS)\bin\delphi_PROJECTICON.ico</Icon_MainIcon>
         <SanitizedProjectName>buildunidata</SanitizedProjectName>
         <Icns_MainIcns>$(BDS)\bin\delphi_PROJECTICNS.icns</Icns_MainIcns>
-        <DCC_UnitSearchPath>C:\keyman\9.0\src\ext\vclzip;c:\keyman\9.0\src\ext\jedi\jwa\branches\2.3\common;c:\keyman\9.0\src\ext\jedi\jwa\branches\2.3\win32api;c:\keyman\9.0\src\ext\dcpcrypt;c:\keyman\9.0\src\ext\dcpcrypt\ciphers;c:\keyman\9.0\src\ext\dcpcrypt\hashes;$(DELPHI)\lib;$(DCC_UnitSearchPath)</DCC_UnitSearchPath>
```

Comment:
I'm guessing some of these were missed in the prior PR, #6694?

Reply:
Invalid paths are ignored by Delphi here. This relates to search paths needing to be updated when I move folders, not when I removed components as in #6694.